.github/ISSUE_TEMPLATE/suggest_wallet.md

```diff
@@ -6,98 +6,192 @@ labels: "wallet :purse:,"
+
+### Security
+
+**Is the wallet code open-sourced?**
+
+<!-- If yes, please provide a direct link to the repository. -->
+
+**What license was the wallet software released under?**
+
+<!-- Please provide information on the software license used for the wallet. -->
+
+**Is your wallet custodial, or non-custodial?**
 
 <!-- Do users have access to their public and/or private keys? If your company/project were to disappear, would users still be able to access their funds? -->
 
 **Please describe the measures taken to ensure the wallet's security and provide documentation wherever possible**
 
-<!-- Please provide a link to a report or repo. If you haven't been audited but think your wallet should be listed anyway, explain here. -->
+<!-- Please provide a link to any security audits or code reports. If you haven't been audited but think your wallet should be listed anyway, explain here. -->
 
-**Does the wallet support layer 2 networks?**
+**Spam protection?**
 
-<!-- Please list which L2 solutions the wallet supports. -->
+<!-- Does the wallet employ any practices to warn users against potential spam (e.g. when interacting with suspicious accounts/contracts)? -->
 
-**Can the wallet be used with arbitrary Ethereum RPC endpoint?**
+### Features
 
-<!-- Does wallet allow user to use the wallet with own Eth client? -->
+**Does the wallet have hardware wallet support?**
 
-**Does the wallet have fiat on-ramps?**
+<!-- Please provide information on how a user can connect a hardware wallet to this wallet. -->
 
-<!-- Does the wallet subsidise transaction fees for all or some wallet activity? -->
+**Does the wallet support WalletConnect?**
 
-**Does the wallet allow users to explore dapps?**
+<!-- Please provide information on if the wallet supports WalletConnect. -->
 
-<!-- Is the wallet integrated with WalletConnect? Does it have a dapp browser? Or can users connect to a dapp using the wallet?  -->
+**Does the wallet support importing Ethereum RPC endpoints?**
 
-**Does the wallet have integrated defi/financial tools?**
+<!-- Please provide documentation on how a user can import an Ethereum RPC into the wallet. -->
 
-<!-- Can users borrow/earn/lend assets directly from a screen in the wallet?  -->
+**Does the wallet support viewing and interacting with NFTs?**
 
-**Can a user withdraw to their card?**
+<!-- Please provide information on the experience of NFTs in the wallet. -->
 
-<!-- Can a user cash out their crypto assets directly to a card or bank account? -->
+**Does the wallet support connecting to Ethereum applications?**
 
-**Does the wallet offer limits protection?**
+<!-- Please provide documentation for how users connect to applications. List examples (ie. connect wallet to dapp, in wallet browser, etc.) -->
 
-<!-- Can a user set a limit for transfers to protect their assets? -->
+**Does the wallet support staking directly?**
```

Comment:
I wonder how important this feature is given that users could just swap for liquid staking tokens?

Reply:
Probably less important. There is still a difference between adding more ETH to the pool, compared to just buying someone elses stake, but I hear ya. I think it's still worth leaving this in.